mod-source-record-manager-server/src/main/java/org/folio/verticle/AbstractConsumersVerticle.java

```diff
@@ -12,6 +12,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import static org.folio.rest.jaxrs.model.DataImportEventTypes.DI_RAW_RECORDS_CHUNK_READ;
```

Comment:
Looks like this might not be used in this file.

Reply:
Thank you. Fixed.